latex2sympy.py

```diff
@@ -486,6 +486,21 @@ def convert_atom(atom):
 
         # return the symbol
         return symbol
+    elif atom.DOLLAR_NUMBER():
+        text = atom.DOLLAR_NUMBER().getText().replace("\\$", "").replace(",", "")
+        try:
+            sr = sympy.Rational(text)
+            return sr
+        except (TypeError, ValueError):
+            return sympy.Number(text)
+    elif atom.PERCENT_NUMBER():
+        text = atom.PERCENT_NUMBER().getText().replace("\\%", "").replace(",", "")
```

Comment:
@builtbywill percentages won't have commas, but it's probably fine to leave this check here.